docs/v0.5/en/faq-and-others/faq.md

```diff
@@ -21,9 +21,9 @@ GreptimeDB supports SQL and can deal with non-time-series data, especially effic
 
 Yes, you can find our Golang SDK [here](https://github.com/GreptimeTeam/greptimedb-client-go).
 
-Currently, we support MySQL protocol, you can check it out on the [user guide](https://docs.greptime.com/user-guide/supported-protocols/mysql).
```

Comment:
These links need to be updated, thanks @tisonkun

Reply:
What does it mean? I checked this link locally, and it works well. To what content you'd like to update?

Reply:
oh my fault, I mean, this change is great 👍